sdk_extensions/logging/support/src/main/java/io/opentelemetry/logging/LoggingBatchExporter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.logging;
+
+import io.opentelemetry.logging.api.LogRecord;
+import java.util.Collection;
+
+/**
+ * A LoggingBatchExporter accepts a batch of records and handles the transmission of those records
+ * to a remote system.
+ */
+public interface LoggingBatchExporter {
```

Comment:
s/LoggingBatchExporter/LogsExporter to be in sync with others exporters. Also move it to a directory called `export`